packages/web/pages/api/pools.ts

```diff
@@ -37,6 +37,6 @@ export default async function pools(req: Request) {
 }
 
 export const config = {
-  runtime: "experimental-edge",
+  runtime: "edge",
```

Comment:
what is this change?

Reply:
They changed it, it's was an experimental feature, but now it's stable. In fact, I inserted it because a warning message appeared suggesting replacing it